full translation of c00 (remaining c00/30/21->32, c00/34, c00/50) (#72)
IllusionMods/HS2-Translation: Translation/en/RedirectedResources/assets/abdata/adv/scenario/c00/50/50/1/translation.txt

// Dumped for Honey Select 2 v1.2.0 by Text Dump v1.4.3
//
//attitude:favor

> 水の抵抗は厄介でしたね。まぁ、それが興奮に繋がっていたのですが……=The water resistance was a bit of a nuisance. Well, I guess that just made it more exciting after all...

Reviewer: ```suggestion
水の抵抗は厄介でしたね。まぁ、それが興奮に繋がっていたのですが……=Moving like that in the water was awkward. Well, I guess that just made it more exciting after all...
```

Author: i went with your suggestion, but it's a bit far from the jp line...  how about we take a bit from the mtl?
"moving like that in the water was a bit tricky. ...." ?

Reviewer: How is it far off? Isn't it literally about resisting/opposing/fighting against the water and how it was bothersome/troublesome to do so.  Putting that into somewhat reasonable conversation english I got `Moving like that in the water was awkward`. I was trying to be true to the original.
